.changeset/small-jars-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+"docs": patch
+---
+
+add section comments in docs for devrel team
```

Comment:
@sarahschwartz I believe the changeset should be empty because no changes were made to the source code, and therefore, we do not want to bump versions.